### ISSUE TITLE
Publish latest CTN updates to vector tiles and feature service

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#7048]

**Comment by andrewshensky** [comment]
@alan-deanda The CTN should be pretty up to date at this point. Let me know if you have some time and we can discuss/work through the vector tile publishing process

**Comment by jaime-mckeown** [comment]
Create a monthly update process for Moped.

**Comment by jaime-mckeown** [comment]
Verify with Alan on fields that are needed.

**Comment by jaime-mckeown** [comment]
Give John head's up on fields included before publishing the update.

**Comment by amenity** [comment]
@jaime-mckeown @johnclary - is this still in progress?

**Comment by jaime-mckeown** [comment]
I'm not sure, @andrewshensky?

**Comment by andrewshensky** [comment]
@amenity I'll move this to Review/QA. I have been working on a new process to update the vector tiles as part of our monthly CTN update process. I figured out a way to update the vector tiles while keeping the service url stable. Due to this, I have created two new vector tile layers that should now serve as the master layers, but they likely need review. I was planning to discuss this with John during geo sprint planning tomorrow.
